src/ops.rs

```diff
@@ -408,8 +409,20 @@ fn op_fetch(
   let url = url.parse::<hyper::Uri>().unwrap();
   let client = http_util::get_client();
 
+  let mut req = hyper::Request::new(hyper::Body::from(&*data));
+  *req.method_mut() = match method {
+    "POST" => hyper::Method::POST,
+    "GET" => hyper::Method::GET,
+    _ => panic!("Invalid method"),
+  };
+  *req.uri_mut() = url.clone();
+  req.headers_mut().insert(
+    hyper::header::CONTENT_TYPE,
+    hyper::header::HeaderValue::from_static("application/json"),
```

Comment:
Content type shouldn't be hardcoded. It should be taken from headers, which at the moment are not passed to Rust.

Reply:
Yeah I know but it was just a quick implementation before I added headers. Will be fixed in next few commits.